src/Directory.Build.targets

```diff
@@ -122,7 +123,10 @@
       <TlbExpAssemblyPaths Include="@(_TlbExpAssemblyPaths->'%(SlashlessPath)')" />
     </ItemGroup>
 
-    <Exec Command="&quot;$([Microsoft.Build.Utilities.ToolLocationHelper]::GetPathToDotNetFrameworkSdkFile('tlbexp.exe'))&quot; $(TlbExpVerbosity) /NOLOGO @(TlbExpAssemblyPaths->'/asmpath:&quot;%(Identity)&quot;', ' ') &quot;$(TargetPath)&quot; /out:&quot;$(TargetDir)$(TargetName).tlb&quot;" />
+    <Error Condition=" '$(TlbExpPath)' == '' "
+           Text="TlbExp was not found. Ensure that you have installed everything from .vsconfig. If you have, please report a bug to MSBuild." />
```

Comment:
`installed everything from .vsconfig` sounds ambiguous. But maybe it's one of those "you know it when you see it" kind of things.

Reply:
After #4676, Visual Studio itself should prompt to install workloads whenever you open one of the solutions, so _hopefully_ this is fairly rare/clear going forward. But if people stumble over it we can change it.